# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../unet/84938ba027730cd1b2fe589b2bcd635df9dbd01b.patch
# Copyright (C) 2009 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

From 84938ba027730cd1b2fe589b2bcd635df9dbd01b Mon Sep 17 00:00:00 2001
From: Alejandro Mery<amery@geeks.cl>
Date: Tue, 17  Feb  2009  17:04:07  +0000
Subject: unet-dhcp: added 'down' mode to udhcpc script

---
diff --git a/unet-udhcpc.in b/unet-udhcpc.in
index 9963cd3..44a3c6d 100644
--- a/unet-udhcpc.in
+++ b/unet-udhcpc.in
@@ -28,6 +28,10 @@ bound)
 		echo "nameserver $x" >> /etc/resolv.conf
 	done
 	;;
+down)
+	ip addr flush dev "$interface"
+	ip link set "$interface" down
+	;;
 leasefail|*)
 	false
 	;;
--
cgit v0.8.1
